float64[] times
string[] names
sensor_msgs/JointState[] states